### PR TITLE
Add spec file with systemd unit file

### PR DESCRIPTION
**Pull request by machacekondra**

### CODE REVIEW

**Comment by machacekondra** [comment]
cc @mwperina

**Comment by machacekondra** [comment]
Hello @pcuzner @jmolmo can you please review/merge if all is OK? Thank you!

**Comment by jmolmo** [comment]
As we already commented in previous occasions, to have an spec file implies make decisions about how is going to  be deployed the service. For example, using gunicorn, using nginx with TLS mutual auth or not, or other web servers. Use a systemd unit file or not ...

I think that is good to have this stuff in the ARS, but is needed to improve our current folder structure and documentation. (To reflect the different possibilities).

Can you make the following changes?
1. Create a "deploy" folder, This folder should contain a subfolder for each deploy configuration. In your case , you should create a "gunicorn" folder where to store your spec file and related systemd files.

2. Use always the "wsgy.py"  file we have in the root folder.

File: runner_service/wsgi.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+"""
+
+# wsgi entry point is only for production servers
+configuration.init(mode='prod')
+
+# The object to be managed by uwsgi
+application = create_app()
```
[the remark this answers is not in the document]
Comment (machacekondra) [reply runner_service/wsgi.py]: This one doesn't call `setup_common_environment`, which is something we don't want in this case.

**Comment by machacekondra** [comment]
Will do.

**Comment by machacekondra** [comment]
@jmolmo Thank you for review, I've updated the PR with your comments..

**Comment by jmolmo** [comment]
LGTM

**Pull request merged**